src/client/visualizations/table/table.scss

```diff
@@ -2,173 +2,64 @@
 
 .table {
-  .sort-arrow {
-    position: absolute;
-    top: 12px;
-    width: 8px;
 
-    &.ascending {
-      transform: rotate(180deg);
-    }
-  }
-
-  .corner {
+  // Used for both row (i.e. body rows) and segments
+  @mixin row-style {
     position: absolute;
-    left: $space-left;
-    top: 0;
-    width: $segment-width;
-    height: $header-height;
-    overflow: hidden;
-    padding: $header-padding-top 0 0 6px;
+    height: $row-height;
     border-bottom: 1px solid $border-extra-light;
+    left: 0;
 
-    .sort-arrow {
-      right: 2px;
-    }
-  }
-
-  .simple-table {
-    .header-cont {
-      left: $space-left + $segment-width;
-
-      .header {
-        height: $header-height;
-      }
-
-      .sort-arrow {
-        right: -15px;
-      }
-    }
-
-    .body-cont {
-      @include unpin-top($header-height);
-      left: $space-left + $segment-width;
-    }
-
-    .row {
-      width: 100%;
+    &.hover {
+      background: $hover;
     }
 
-    .segments-cont {
-      position: absolute;
-      left: $space-left;
-      top: $header-height;
-      bottom: 0;
-      width: $segment-width;
-      overflow: hidden;
+    &.selected {
+      background: $hover;
     }
 
-    .highlight-cont {
-      position: absolute;
-      left: $space-left;
-      right: 0;
-      top: $header-height;
-      bottom: 0;
-      overflow: hidden;
-
-      .highlighter {
-        right: 0;
-        height: $row-height + 1;
-        margin-top: -1px;
-        border: 1px solid $hover;
-
-        &:after {
-          @include unpin-full(-1px);
-          content: '';
-          border: 1px dashed $highlight-border;
-        }
+    &.not-selected {
+      &:after {
+        @include pin-full;
+        background: $white;
+        opacity: 0.5;
+        content: '';
       }
     }
   }
 
-  .vertical-scroll-shadow {
+  .corner {
     position: absolute;
     left: $space-left;
     top: 0;
-    bottom: 0;
     width: $segment-width;
-    box-shadow: 4px 0 4px -3px hsla(0, 0, 0, 0.2);
-  }
-
-  .highlight-controls {
-    position: absolute;
-    right: 20px;
-  }
-
-  .segment,
-  .row {
-    position: absolute;
-    height: $row-height;
+    height: $header-height;
+    overflow: hidden;
+    padding: $header-padding-top 0 0 6px;
     border-bottom: 1px solid $border-extra-light;
 
-    &.hover {
-      background: $hover;
-    }
-
-    &.selected {
-      background: $hover;
-    }
-
-    &.nest0 {
-      background: $background-lightest;
-    }
-  }
-
-  .row {
-    left: 0;
-  }
-
-  .not-selected {
-    &:after {
-      @include pin-full;
-      background: $white;
-      opacity: 0.5;
-      content: '';
+    .sort-arrow {
+      right: 2px;
     }
   }
 
-  .measure-name {
-    padding-top: $header-padding-top;
+  .simple-table {
+    @import './header';
+    @import './segments'; // first, left column of the table
+    @import './body'; // actual values
+    @import './post-body'; // v-scroll shadow, highlighter...
```

Comment:
OMG yes. So clean! did not know you could do this. It was not possible last time I checked (which was like 3 years ago)